pilot/test/test_copytools_rucio.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Authors: Pavlo Svirin <pavlo.svirin@gmail.com>
+
+import unittest
+import os
+
+# from pilot.control.job import get_fake_job
+# from pilot.info import JobData
+from pilot.info.filespec import FileSpec
+from pilot.util.tracereport import TraceReport
+
+
+def check_env():
+    """
+    Function to check whether rucio copytool is loaded correctly.
+    To be used to decide whether to skip some test functions.
+    :returns True: if rucio copytool is available. Otherwise False.
+    """
+    aval = False
```

Comment:
Tomas, here you can simply try to import Rucio client to decide if rucio environment is available (in order to run tests at proper machine).

Reply:
Ok. Cool. There is the problem with the logs at the moment, that has to be solved. Namely, that this version of rucio copytool produces empty logs.